### PR TITLE
Python tests ci pipeline

### PR DESCRIPTION
**Pull request by Mryashbhardwaj**

### CODE REVIEW

File: ext/scheduler/airflow2/tests/Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 all: build
 
 test:
-	@python -m unittest discover .
```
Comment (sravankorumilli) [inline ext/scheduler/airflow2/tests/Makefile:4]: Lets move this target into the base one as it becomes hard for discoverability & no one runs these tests

File: .github/workflows/scheduler-resources-test.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: test optimus resources
```
Comment (sravankorumilli) [inline .github/workflows/scheduler-resources-test.yml:1]: why not integrate in make test-ci target, and we don't need another workflow for running unit tests.

**Review by sravankorumilli — CHANGES_REQUESTED** [review]

File: .github/workflows/scheduler-resources-test.yml (hunk 2 above)
Comment (Mryashbhardwaj) [reply .github/workflows/scheduler-resources-test.yml]: sure

Comment (Mryashbhardwaj) [reply .github/workflows/scheduler-resources-test.yml]: sure

**Pull request merged**